float64 correlation
map_localiser/MatchedChainElement[] elements